### DISCUSSION TITLE
Data Importer Duplicate of transaction

### DISCUSSION THREAD

**Discussion by fedex03** [firefly-iii/firefly-iii#7230] [Firefly III]
Hi there,

I deleted some transactions via the GUI using the "Action" button, but when I try to reimport these transaction, FireFIly Data Importer show the error "ine #0: transactions.0.description: Duplicate of transaction.....".

I deleted this transaction because they are duplicated by my bank because the description filed has changed when I made a new Excel export.

Is there a way to truly delete a transaction?

Thank you very much for your excelent work!!!!

**Comment by JC5** [comment]
If you're sure you wish to reimport transactions, press the **Purge** button in your Profile (under `/profile`).

**Reply by fedex03** [reply]
Hi @JC5,

I already delete these 10 transaction manually from the GUI, but I cannot import them again from my CSV file.

The purge button is intended to delete ALL transaction. Am I righth?

It seems that some trasnactions are appended somewhere.
Not shown in the GUI but DataImport finds them duplicated.

Thank you.

**Reply by JC5** [reply]
> The purge button is intended to delete ALL transaction. Am I righth?

I thought the text was clear enough:  "Purging" means "deleting that which is already deleted". In normal circumstances, Firefly III deletes nothing permanently. It just hides it. The button below deletes all of these previously "deleted" records FOREVER.

**Comment by fedex03** [comment]
Hi @JC5,

you were very clear, but when I opened Profile I cannot find a specific "Purge" button. 
The italian translation means "Remove all transaction".
So I worried about make some "disaster"

![immagine](https://user-images.githubusercontent.com/16340343/225267833-80b7c73a-81f8-4724-af84-12cd439f3225.png)

Thank you

**Reply by JC5** [reply]
Aha. Please upgrade to 6.0.4, there you will find the necessary button (orange color)

**Comment by fedex03** [comment]
Thank you! 
After the update I found the "Purge" Button.

Thank you very much for the help!
